common.yml

```diff
@@ -0,0 +1,22 @@
+services:
+  # Common configuration for all of our core uservices
+  oada-uservice:
+    build:
+      context: ./oada
+    restart: unless-stopped
+    networks:
+      - startup_net
+      - kafka_net
```

Comment:
not all the services need to be on all the networks.  proxy doesn't need to be on kafka or arango, I don't think auth needs kafka (could be wrong, but I don't think so).  Should probably leave network selection up to the service.

Reply:
Only services that merge this anchor would have those networks. Most internal services do these both of these.

Reply:
Isn't the point of the anchor in the monoimage setup that all the services extend from it to centralize/reuse a consistent config?  It seems like the anchor should contain the least common denominator rather than including networks that not all containers need.

Reply:
What OADA micro-service in this repo doesn't need kafka or startup? Also, this is how it was ...